### PR TITLE
Add consoles for haproxy and cloudwatch.

### PR DESCRIPTION
**Pull request by brian-brazil**
Only display jobs in the LHS if they're present.

### CODE REVIEW

File: consoles/aws_elasticache.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{{ template "head" . }}
+
+{{template "prom_right_table_head"}}
```
Comment (juliusv) [inline consoles/aws_elasticache.html]: missing spaces within curly braces (incosistent with rest)

File: consoles/aws_elasticache.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<tr>
+  <td>Cache Items</td>
+  <td>{{ template "prom_query_drilldown" (args (printf "aws_elasticache_curr_items_average{job='aws_elasticache',cache_cluster_id='%s'}" .Labels.cache_cluster_id) "" "humanize") }}</td>
+</tr>
+<tr>
+  <td>Freeable Memory</td>
+  <td>{{ template "prom_query_drilldown" (args (printf "aws_elasticache_freeable_memory_average{job='aws_elasticache',cache_cluster_id='%s'}" .Labels.cache_cluster_id) "B" "humanize1024") }}</td>
+</tr>
+{{ end }}
+{{template "prom_right_table_tail"}}
```
Comment (juliusv) [inline consoles/aws_elasticache.html]: dito and below

**Comment by juliusv** [comment]
I haven't tried it out, but LGTM besides whitespace consistency issues.

**Comment by brian-brazil** [comment]
Fixed.

**Pull request merged**